doctr/models/utils.py

```diff
@@ -140,7 +142,9 @@ def conv_sequence(
     kwargs['use_bias'] = kwargs.get('use_bias', not(bn))
     # Add activation directly to the conv if there is no BN
     kwargs['activation'] = activation if not bn else None
-    conv_seq = [layers.Conv2D(out_channels, padding=padding, kernel_initializer=kernel_initializer, **kwargs)]
+    conv_seq = [
+        layers.Conv2D(out_channels, strides=strides, padding=padding, kernel_initializer=kernel_initializer, **kwargs)
+    ]
```

Comment:
Since the kwargs are passed to the Conv2D, we don't need to add it (simply passing `strides=...` when calling `conv_sequence`)